setup.py

```diff
@@ -87,6 +90,58 @@ def download_file(URL, dest_dir, local_file, num_retries=4):
     return return_code
 
 
+def app_available(app_name):
```

Comment:
Yes, all of the functions for downloading/installing dependency tools (Vsearch, SortMeRNA, MAFFT?) were copied from QIIME's setup.py.